beacon-chain/p2p/peers/status.go

```diff
@@ -800,134 +811,87 @@ func (p *Status) BestNonFinalized(minPeers int, ourHeadEpoch primitives.Epoch) (
 // bad response count. In the future scoring will be used
 // to determine the most suitable peers to take out.
 func (p *Status) PeersToPrune() []peer.ID {
```

Comment:
This is a sensitive function, can you refactor this in a separate PR ?

Reply:
Done in https://github.com/prysmaticlabs/prysm/pull/14460.